tests/common.go

```diff
@@ -12536,3 +12554,235 @@ func GetAllMultipathDevicesPresent(n *node.Node) ([]MultipathDevices, error) {
 
 	return multiPathDevs, nil
 }
+
+// GetMultipathDeviceIDsOnNode returns List of all Multipath Devices on Node
+func GetMultipathDeviceIDsOnNode(n *node.Node) ([]string, error) {
+	multiPathDev := []string{}
+	multiPathDevices, err := GetAllMultipathDevicesPresent(n)
+	if err != nil {
+		return nil, err
+	}
+	for _, eachMultipathDev := range multiPathDevices {
+		multiPathDev = append(multiPathDev, eachMultipathDev.DevId)
+	}
+	return multiPathDev, nil
+}
+
+// CreateNameSpace Creates new NameSpace
```

Comment:
there is already a function with same name in common.go , can we change the name of the function..